src/image-button/image-button.style.tsx

```diff
@@ -21,29 +21,45 @@ export const Main = styled.button`
 
     img {
         width: 3.5rem;
-    }
-
-    ${MediaQuery.MaxWidth.tablet} {
-        padding: 1.5rem 0rem;
+        height: 3.5rem;
+        object-fit: contain;
     }
 
     ${MediaQuery.MaxWidth.mobileL} {
-        padding: 1rem 0rem;
+        padding: 0.5rem 0.25rem;
+        gap: 0.25rem;
     }
 
-    ${MediaQuery.MinWidth.tablet} {
-        :focus:active {
-            box-shadow: ${DesignToken.ElevationBoxShadow};
-            ${TextStyleHelper.getTextStyle("H4", "bold")}
-        }
+    ${(props) => {
+        if (props.$error) {
```

Comment:
is there a selected + error styling also?

Reply:
yes, there can be, updated the PR.